server/src/database/migrations/sql/1_initial_schema.up.sql

```diff
@@ -85,7 +85,7 @@ create table board_sessions
 (
     "user"       uuid         not null references users ON DELETE CASCADE,
     "board"      uuid         not null references boards ON DELETE CASCADE,
-    "show_hidden_columns" boolean not null DEFAULT true,
+    "show_hidden_columns" boolean not null DEFAULT false,
```

Comment:
The initial migration script is in the changelist because (blame on me) I did change it 16 days ago to alter the default value. But now I've noticed that it doesn't work so I changed it back to the original state and added a new script